Integrations/Flashpoint/Flashpoint.py

```diff
@@ -0,0 +1,1321 @@
+from typing import Dict, Any
```

Comment:
move this import after `"""IMPORTS"""`

Reply:
```
import demistomock as demisto
from CommonServerPython import *
from CommonServerUserPython import *
```
these 3 lines should be at the top of your code

Reply:
updated the imports, as the first and the third is not used in our integration, removed them